components/MainView.js

```diff
@@ -17,7 +17,7 @@ window.onload = function () {
             
             const raw = localStorage.getItem('itemList') || ""
             const itemList = this.shuffle(raw.split(",").map(x => x.trim()).filter(x => x !== ""))
-            this.setState({ itemList })
+            this.setState({ itemList, displayText: `Press Enter to Begin\nPress E to Edit List (${itemList.length || 0})` })
```

Comment:
Isn't the ` || 0` is redundant ? Since the `length` can not be anything but 0 or positive number.

Reply:
If itemList is evaluated to be undefined then the itemList.length would be evalusted to be undefined also

Reply:
An example of this could be if a user entered unexpected characters, potentially some non ascii characters that could malform the data and return null for the itemList.